lib/smart_city/dataset.ex

```diff
@@ -44,7 +44,7 @@ defmodule SmartCity.Dataset do
       "protocol": "",       // List of protocols to use. Defaults to nil. Can be [http1, http2]
       "authUrl": "",
       "sourceFormat": "",
-      "sourceType": "",     // remote|stream|batch
+      "sourceType": "",     // remote|stream|ingest|host
```

Comment:
When we start consuming this library we will likely need to update migration if we want it to stay up to date.

Reply:
Agreed